.github/workflows/codeql.yml

```
@@ -1,17 +1,101 @@
+      # Query set to use when analyzing the source code (e.g. default, security-extended, security-and-quality)
+      analysis-query: security-and-quality
+      
 
 on:
   # Runs on pull requests and on pushes to main (in order to keep the regular scanning by GitHub working)
   pull_request:
   push:
     branches: [main]
 
+permissions:
+  pull-requests: read
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 2

(indentation)